floop-data-export.py

```diff
@@ -38,7 +40,7 @@
 
     for convo in conversations:
         convo_entries = convo.reference.collection(
-            'Messages').order_by(u'Date_Submitted').limit(1).get()
+            'Messages').order_by(u'Date_Submitted').get()
 
         for entry in convo_entries:
             cList.append(entry.get("Text").strip())
```

Comment:
Based on [Dan's instructional comment in the Tuesday Repo's similar issue](https://github.com/North-Seattle-College/ad440-winter2022-tuesday-repo/issues/54#issuecomment-1027571054), I _think_ this is the kind of modification that would be needed:
![image](https://user-images.githubusercontent.com/49133101/153662061-8e42887f-e456-4d1b-a3c8-31614e4d0b6f.png)